test/storage/tile_compression_test.cpp

```diff
@@ -0,0 +1,181 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// tile_compression_test.cpp
+//
+// Identification: test/storage/tile_compression_test.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "common/harness.h"
+
+#include "storage/tile.h"
+#include "storage/dictionary_encoding_tile.h"
+#include "storage/tile_group.h"
+#include "storage/tuple_iterator.h"
+#include "type/value_factory.h"
+
+namespace peloton {
+namespace test {
+
+//===--------------------------------------------------------------------===//
+// Tile Tests
+//===--------------------------------------------------------------------===//
+
+class TileCompressionTests : public PelotonTest {};
+
+TEST_F(TileCompressionTests, BasicTest) {
```

Comment:
I'm failing this test:
Assertion failed: ((column_offset < schema.GetLength())), function SetValueFast, file /Users/matt/git/dictencode/src/storage/tile.cpp, line 180.